srcs/parser/map_name_extention_checker.c

```diff
@@ -0,0 +1,35 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   extention.c                                        :+:      :+:    :+:   */
```

Comment:
extension?

Reply:
quite a long name for the file dont you think? check_infile_extension ? check_file_extension ?

Reply:
Can't agree more xD